log4j-api/src/main/java/org/apache/logging/log4j/message/DefaultFlowMessageFactory.java

```diff
@@ -51,17 +52,22 @@ public DefaultFlowMessageFactory() {
      * @param exitText the text to use for trace exit, like {@code "Exit"}.
      */
     public DefaultFlowMessageFactory(final String entryText, final String exitText) {
+        this(entryText, exitText, createDefaultMessageFactory());
+    }
+
+    public DefaultFlowMessageFactory(final MessageFactory messageFactory) {
```

Comment:
Where do we use this?

Reply:
I planned to use it in Log4j Core 3.x, so that the same `MessageFactory` is used by `DefaultFlowMessageFactory` and `AbstractLogger`.

Thinking about it, I might have been overzealous in #1171, when I introduced the `messageFactory` field. I am removing the field and hardcoding it to `ParameterizedMessageFactory.INSTANCE`.